test/Microsoft.TestPlatform.Utilities.UnitTests/MSTestSettingsUtilitiesTests.cs

```diff
@@ -80,14 +75,18 @@ public void ImportShouldEmbedTestSettingsInformation()
             xmlDocument.LoadXml(defaultRunSettingsXml);
             var finalxPath = MSTestSettingsUtilities.Import(
                 "C:\\temp\\r.testsettings",
-                xmlDocument,
-                Architecture.X86,
-                FrameworkVersion.Framework45);
+                xmlDocument);
 
             var finalSettingsXml = finalxPath.CreateNavigator().OuterXml;
 
             var expectedSettingsXml =
-                "<RunSettings>\r\n  <MSTest>\r\n    <SettingsFile>C:\\temp\\r.testsettings</SettingsFile>\r\n    <ForcedLegacyMode>true</ForcedLegacyMode>\r\n  </MSTest>\r\n  <RunConfiguration></RunConfiguration>\r\n</RunSettings>";
+                "<RunSettings>\r\n" +
```

Comment:
nit: In order to make the string as a whole more readable it would be nice to remove `\r\n` and instead do `string.Join` with `Environment.NewLine`. Also using verbatim strings (`@""`) should help with readability.

Reply:
You are right, I just used regex to split it. It's merged already, next time :)

Reply:
Will definitely face it when fixing tests to run on linux.